client/src/modules/inventory/list/templates/isbrocken.cell.html

```diff
@@ -0,0 +1,12 @@
+<div class="ui-grid-cell-contents">
+  <span 
+    class="text-danger text-right"
+    ng-show='row.entity.is_brocken==1'
+    translate>
+    FORM.LABELS.YES
+  </span>
+  <span
```

Comment:
You could save some space by moving the "text-right" class to the parent `<div>`.